package.json

```diff
@@ -169,12 +169,12 @@
     "workerjs": "github:jasonLaster/workerjs"
   },
   "lint-staged": {
-    "bin/*.js": ["yarn lint", "git add"],
-    "src/*.js": ["yarn lint", "git add"],
-    "src/*/*.js": ["yarn lint", "git add"],
-    "src/*/!(mochitest)**/*.js": ["yarn lint", "git add"],
-    "src/*/!(mochitest)*/**/*.js": ["yarn lint", "git add"],
-    "packages/**/src/*.js": ["yarn lint", "git add"]
+    "bin/*.js": ["yarn lint --findRelatedTests", "git add"],
```

Comment:
Long story short, lint-staged takes the staged files and makes them a space separated list. Jest doesn't understand this by default. This flag makes it understand.